extensions/wikia/Insights/specials/UnconvertedInfoboxesPage.class.php

```diff
@@ -80,6 +81,8 @@ public function recache( $limit = false, $ignoreErrors = true ) {
 			}
 		}
 
+		wfRunHooks( 'UnconvertedInfoboxesQueryRecached' );
```

Comment:
For me hard to say what's the purpose of this code and functionality in context of Dat 3193.